contracts/modules/maker/MakerV2Base.sol

```diff
@@ -77,7 +77,7 @@ contract MakerV2Base is BaseModule, RelayerModule, OnlyOwnerModule {
         BaseWallet _wallet,
         uint256 _amount
     )
-        public
+        external
```

Comment:
why did you make that change? Isn't it better for the constructor of a base contract to be `public` so it can be called in the constructor of a contract that extends it?

Reply:
Yes absolutely. Not sure where this change came from and why I don't see it in my local repo. Weird.

Reply:
Oh I get it, the change is not on the constructor, it's on `swapSaiToDai()` which didn't need to be made public any more after we decided not to auto-convert SAI to DAI for debt repayment.